package.json

```diff
@@ -159,7 +159,7 @@
     "react-native-localize": "^2.2.6",
     "react-native-modal": "^13.0.0",
     "react-native-onyx": "2.0.64",
-    "react-native-pager-view": "6.2.3",
+    "react-native-pager-view": "^6.3.4",
```

Comment:
```suggestion
    "react-native-pager-view": "6.3.4",
```

Reply:
This makes no difference currently (since 6.3.4 is the latest version as of now), but _could_ in the future (probably not, but let's be safe)